From 49a1fb77700ef01af7186ba427262fa710b2f3c6 Mon Sep 17 00:00:00 2001
From: Son Lam <son.lam.ym@renesas.com>
Date: Mon, 15 Jun 2020 17:33:30 +0700
Subject: [PATCH] gzip: fseterr: adjust to glibc 2.28 libio.h removal

Unfortunately, _IO_ftrylockfile got removed from glibc/2.28 and
__GNU_LIBRARY__ is 6, so glibc is not a GNU libc.

Signed-off-by: Son Lam <son.lam.ym@renesas.com>
---
 lib/fseterr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/fseterr.c b/lib/fseterr.c
index 5aaa51f..d0ba48b 100644
--- a/lib/fseterr.c
+++ b/lib/fseterr.c
@@ -29,7 +29,7 @@ fseterr (FILE *fp)
   /* Most systems provide FILE as a struct and the necessary bitmask in
      <stdio.h>, because they need it for implementing getc() and putc() as
      fast macros.  */
-#if defined _IO_ftrylockfile || __GNU_LIBRARY__ == 1 /* GNU libc, BeOS, Haiku, Linux libc5 */
+#if defined _IO_EOF_SEEN || __GNU_LIBRARY__ == 1 /* GNU libc, BeOS, Haiku, Linux libc5 */
   fp->_flags |= _IO_ERR_SEEN;
 #elif defined __sferror || defined __DragonFly__ || defined __ANDROID__
   /* FreeBSD, NetBSD, OpenBSD, DragonFly, Mac OS X, Cygwin, Android */
-- 
2.7.4

